inception-project/inception
# #1127 Improve Permanent AnnotationPage Error messages

A layer-recommender mismatch produces error-messages on the `AnnotationPage RecommendationSideBar`. These have to be clicked to close every time by the user and intrude on other components in the user's view.

[Update 2019-05-03]: After #808, the learning curve chart no longer triggers error-messages.

**What's in the PR**
Use a different solution than the Feedback Panel for error messages with longer duration, for:
* Layer-recommender mismatch on the annotation page (see https://github.com/inception-project/inception/issues/542)

**How to test manually**
* Configure your project in such a way that the respective errors occur (see respective Issue description)
* Open the annotation page and the recommender sidebar.

**Automatic testing**
* no unit tests
